extensions/vscode/src/commands/stop-dev-server.ts

```diff
@@ -0,0 +1,190 @@
+import {
+  QuickInputButton,
+  QuickPickItem,
+  QuickPickItemKind,
+  commands,
+  window,
+} from "vscode";
+import { isDartFrogCLIInstalled, suggestInstallingDartFrogCLI } from "../utils";
+import {
+  DartFrogApplication,
+  DartFrogApplicationRegistry,
+  DartFrogApplicationRegistryEventEmitterTypes,
+  DartFrogDaemon,
+  StopDaemonRequest,
+} from "../daemon";
+
+/**
+ * Stops a Dart Frog development server on the current workspace.
+ *
+ * This command can be launched from the command palette.
+ *
+ * The user requires to have Dart Frog CLI installed in order to run this
+ * command. If they opt out of installing Dart Frog CLI, the command will
+ * run at their own risk.
+ *
+ * If there are no running servers or the daemon is not ready, the user will
+ * be prompted to start a server and this command will not run.
```

Comment:
I wonder if this command shouldn't just be a no op in this case. If you want to stop a server, but none is running, what is the reasoning for starting it?

Reply:
It is somewhat of a no-op when there are no servers running (it does nothing). I see it as a bad UX if no feedback is given when the command is executed and that's why I do prefer showing at least some information that there are no servers running.

I have updated the documentation since it might have been ambiguos before.